app/utils/achievementConstants.ts

```diff
@@ -128,6 +128,32 @@ const AchievementsInfo: Record<string, AchievementData[]> = {
       hidden: false,
     },
   ],
+  penalty_period: [
+    {
+      name: 'Pliktoppfyllende',
+      description: 'Gått 1 år uten prikk',
+      rarity: 0,
+      hidden: false,
+    },
+    {
+      name: 'Englebarn',
+      description: 'Gått 2 år uten prikk',
+      rarity: 1,
+      hidden: false,
+    },
+    {
+      name: 'Eksemplarisk',
+      description: 'Gått 3 år uten prikk',
+      rarity: 4,
```

Comment:
This could probably be rarity 5 easily - only 22 users would qualify as it stands now.
Level 1 "Englebarn" should probably be level 3, 133 users qualify.